staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -1024,6 +1098,14 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 		waitGroup:           &utilwaitgroup.SafeWaitGroup{},
 	}
 
+	currentStorageVersion := "vOther"
+	if previousStorageVersion, ok := r.previousCustomStorage.Load(crd.UID); ok && previousStorageVersion == currentStorageVersion {
```

Comment:
I am also having a hard time to understand when would previousSV ever == currentSV